### PR TITLE
Add initial test coverage for native questions embedding

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING CI && PENDING REVIEW

### What does this PR accomplish?
- Adds initial set of tests for the native questions with complex parameters in the embedding context

### Screenshots:
image

Disabled filters
image

Enabled filters
image

### CODE REVIEW

File: frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  });
+
+  it("should display and work with enabled parameters while hiding the locked one", () => {
+    setParameter("Order ID", "Editable");
+    setParameter("Created At", "Editable");
+    setParameter("Total", "Locked");
+    setParameter("State", "Editable");
+    setParameter("Product ID", "Editable");
+
+    // We must enter a value for a locked parameter
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js:127]: Isn't this strange given the fact that we already set the default value for this filter?

File: frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
+import { restore, popover, filterWidget } from "__support__/e2e/cypress";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { ORDERS, PEOPLE, PRODUCTS } = SAMPLE_DATABASE;
+
+const query = `
+SELECT orders.id, orders.product_id, orders.created_at AS production_date, orders.total, people.state, people.name, people.source
+FROM orders LEFT JOIN people ON orders.USER_ID = PEOPLE.id
```
Comment (flamber) [inline frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js:8]: Just consistency
```suggestion
FROM orders LEFT JOIN people ON orders.user_id = people.id
```

**Review by flamber — APPROVED** [review]

File: frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js (hunk 1 above)
Comment (flamber) [reply frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js:127]: No, expected. When you set to "Locked", then it basically means that the embedding will be defining the value and it has to be defined, it's non-optional.

File: frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js (hunk 2 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js]: Good catch, thank you.

**Pull request merged**